integration-tests/hibernate-search-orm-elasticsearch-outbox-polling/pom.xml

```diff
@@ -50,6 +50,11 @@
             <artifactId>quarkus-junit5</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-junit5-internal</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
This is suspicious... integration tests should be like an application, so shouldn't need to rely on "internal" modules?

Did you try putting these tests in `extensions/hibernate-search-orm-elasticsearch-outbox-polling/deployment` instead?

Reply:
This is apparently still relevant... maybe?

Reply:
> ohh and there are more changes in main now 😁 let me rebase ....

And there are some more now... sorry.